chpl/chpl-service/src/main/java/gov/healthit/chpl/validation/surveillance/SurveillanceValidator.java

```diff
@@ -349,4 +355,26 @@ public void validateSurveillanceNonconformities(Surveillance surv) {
 			}
 		}
 	}
+	
+	public void validateSurveillanceAuthority(Surveillance surv) {
```

Comment:
Really? When I review the requirements in OCD-1180 and OCD-1250, I think more checks are needed.